internal/errors/errors.go

```diff
@@ -1,57 +1,76 @@
 // Package errors provides errors with codes and prefixes.
 package errors
 
-import "strconv"
+import (
+	"errors"
+	"fmt"
+	"strconv"
+)
 
-// capnpError holds a Cap'n Proto exception.
-type capnpError struct {
-	typ    Type
-	prefix string
-	msg    string
+// Error holds a Cap'n Proto exception.
+type Error struct {
+	Type   Type
+	Prefix string
+	Cause  error
 }
 
 // New creates a new error that formats as "<prefix>: <msg>".
 // The type can be recovered using the TypeOf() function.
-func New(typ Type, prefix, msg string) error {
-	return &capnpError{typ, prefix, msg}
+func New(typ Type, prefix, msg string) Error {
+	return Error{typ, prefix, errors.New(msg)}
 }
 
-func (e *capnpError) Error() string {
-	if e.prefix == "" {
-		return e.msg
+func (e Error) Error() string {
+	if e.Prefix == "" {
+		return e.Cause.Error()
 	}
-	return e.prefix + ": " + e.msg
+
+	return fmt.Sprintf("%s: %v", e.Prefix, e.Cause)
 }
 
-func (e *capnpError) GoString() string {
-	return "errors.New(" + e.typ.GoString() + ", " + strconv.Quote(e.prefix) + ", " + strconv.Quote(e.msg) + ")"
+func (e Error) Unwrap() error { return e.Cause }
+
+func (e Error) GoString() string {
+	return fmt.Sprintf("errors.Error{Type: %s, Prefix: \"%s\", Cause: fmt.Errorf(\"%v\")}",
+		e.Type.GoString(),
+		e.Prefix,
+		e.Cause)
```

Comment:
Is there a reason these are no longer quoted?

Reply:
Still waiting on a response to this comment; the old version properly escapes the strings, why was this changed?